app/hotels/src/singleHotel/header/Header.js

```diff
@@ -34,13 +33,20 @@ export type Props = {
 export class Header extends React.Component<Props> {
   openGallery = () => {
     const { hotel } = this.props;
-    const photosEdges = idx(hotel, _ => _.photos.edges) || [];
-    const images = photosEdges.map(edge => ({
-      key: idx(edge, _ => _.node.id) || '',
-      lowRes: idx(edge, _ => _.node.lowResUrl) || '',
-      highRes: idx(edge, _ => _.node.highResUrl) || '',
-    }));
-    const hotelName = idx(hotel, _ => _.name);
+    const photosEdges = hotel?.photos?.edges ?? [];
+
+    const images = photosEdges.map(edge => {
```

Comment:
Is it the second time we write this logic? Should we extract it?

Reply:
It is not the exact same, this is a connection, the other one is not. 

I could extract it, but then I would have to map this connection, and then pass it to the extracted function to have it mapped yet again 🤷‍♂️

Reply:
Okay, then no need